README.md

```diff
@@ -1 +1,24 @@
 # Slurm rocks
+
+The `slurm-rocks` repository is a collection of files to build rocks for the upstream
+Slurm services.
+
+## Getting started
+
+To get started with the project, you must have [Rockcraft] and [Just] installed for
+packing the rocks, [Skopeo] for copying the generated OCI images to your local Docker registry, and
+finally [Docker] for running the images.
+
+```bash
+just pack # Packs all the rocks.
+just import # Imports all the rocks into your local Docker registry.
+docker run --rm -d --name slurmctld slurmctld:latest # Runs the slurmctld rock with Docker.
+```
+
+A more extensive usage example can be seen on the (example)[./example] directory, which sets up all
+the Slurm services into a simple cluster.
+
+[Rockcraft]: https://documentation.ubuntu.com/rockcraft
+[Just]: https://github.com/casey/just
+[Skopeo]: https://github.com/containers/skopeo
+[Docker]: https://www.docker.com
```

Comment:
[nit]: Going off of what we have previously done, typically in our READMEs we refer to `just` in the lowercase. Also, look at the docs for `skopeo` on GitHub, it looks like it's preferred to refer to `skopeo` in lowercase as well. I think Rockcraft and Docker are fine capitalized since that's how their referred to in their respective docsets.

```suggestion
[Rockcraft]: https://documentation.ubuntu.com/rockcraft
[just]: https://github.com/casey/just
[skopeo]: https://github.com/containers/skopeo
[Docker]: https://www.docker.com
```

Reply:
Fixed